src/modules/accelerometer.cpp

```diff
@@ -345,7 +352,7 @@ namespace Modules
             {
                 float3 nacc = acc / accMag; // normalize
                 float bestDot = -1000.0f;
```

Comment:
I'm being picky here, but since we spend sometime figuring out why bestDot is initialized as such, a comment to remind us next time what's going is welcome :)

For example:
```
float bestDot = -1.1f; // Note: it should be -1 as we're comparing this value with the dot product
                       //       of 2 normalized vectors, but because of floating point limited
                       //       precision the dot product may return a value slightly lower than -1
```